### PR TITLE
Updated styling for consistency

### PR DESCRIPTION
**Pull request by UAMS16**
image
image

### CODE REVIEW

File: cegs_portal/search/templates/search/v1/experiments.html [hunk 1]
```diff
@@ -58,19 +67,33 @@
...
-                <option value="sources-targets" selected>Tested Elements and Genes</option>
-            </select></div>
-        <ul class="list-disc mt-5">
-            {% for expr in experiments %}
-            <li>{{ expr.name }} <a class="link" href="{% url 'search:experiment' expr.accession_id %}"><svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="currentColor" class="svg-link-arrow bi bi-arrow-up-right-square-fill inline" viewBox="0 0 16 16">
-                <path fill-rule="evenodd" d="M14 0a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2zM5.904 10.803 10 6.707v2.768a.5.5 0 0 0 1 0V5.5a.5.5 0 0 0-.5-.5H6.525a.5.5 0 1 0 0 1h2.768l-4.096 4.096a.5.5 0 0 0 .707.707z"/>
-            </svg></a></li>
-            {% endfor %}
-        </ul>
+        <table class="min-w-full">
```
Comment (tncowart) [inline cegs_portal/search/templates/search/v1/experiments.html:70]: I don' think this should be a table. Or, if you'd like it to be, only the list of experiments should be in the table, not the intersection selector line. Also, was the asterisk added on purpose?

**Review by tncowart — CHANGES_REQUESTED** [review]

File: cegs_portal/search/templates/search/v1/experiments.html (hunk 1 above)
Comment (UAMS16) [reply cegs_portal/search/templates/search/v1/experiments.html]: Removed from table. Asterisk was added

Comment (tncowart) [reply cegs_portal/search/templates/search/v1/experiments.html]: Why did you add the asterisk?

Comment (UAMS16) [reply cegs_portal/search/templates/search/v1/experiments.html]: Deleted

**Review by tncowart — APPROVED** [review]

**Pull request merged**